PROTOCOL.md

```diff
@@ -1933,7 +1933,9 @@ The request body should be a JSON string containing the following optional field
 
 - **timestamp** (type: String, optional): an optional timestamp string in the [Timestamp Format](#timestamp-format),. If set, will return files as of the table version corresponding to the specified timestamp. This is only supported on tables with history sharing enabled.
 
-- **startingVersion** (type: Long, optional): an optional version number. If set, will return all data change files since startingVersion, including historical metadata if seen in the delta log.
+- **startingVersion** (type: Long, optional): an optional version number. If set, will return all data change files since startingVersion, inclusive, including historical metadata if seen in the delta log.
+
+- **endingVersion** (type: Long, optional): an optional version number, only used if startingVersion is set. If set, will return all data change files until endingVersion, inclusive, including historical metadata if seen in the delta log.
```

Comment:
What would be the behavior if a server doesn't support `endingVersion`? The server would still return the entire history? Should we document that the client needs to handle this?

Reply:
updated.